package.json

```diff
@@ -8,6 +8,7 @@
   "license": "GPL-3.0",
   "dependencies": {
     "irc": "^0.5.2",
+    "jsonfile": "^3.0.0",
```

Comment:
@ananyo2012 If you like, I could do it by using `fs` and put the code inside a function in `utils`.